addons/pos_restaurant/static/src/js/Screens/ProductScreen/ActionpadWidget.js

```diff
@@ -38,10 +39,28 @@ patch(ActionpadWidget.prototype, "point_of_sale.ActionpadWidget", {
         }
     },
     get highlightPay() {
+        let countUnit = 0;
+        if (this.currentOrder.orderlines) {
+            this.currentOrder.orderlines.forEach((line) => (countUnit += line.get_quantity()));
```

Comment:
Extract a function that is perhaps called `hasQuantity` (or `qtyGTZero`) that takes an order. Then call that to compute the condition that will be added in this `highlightPay` getter. It will be something like:

```js
hasQuantity(order) {
  if (!order) {
    return false;
  }
  return order.orderlines.reduce((totalQty, line) => totalQty + line.get_quantity(), 0);
}

get highlightPay() {
  return (
this._super(...arguments) &&
            this.pos.globalState.printers_category_ids_set.size &&
            !this.currentOrder.hasChangesToPrint()
            !this.currentOrder.hasChangesToPrint() &&
            hasQuantity(this.currentOrder)
  )
}
```

You'll also be able to get rid of the runbot error.